packages/flutter/test/material/user_accounts_drawer_header_test.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```

Comment:
You should include simple tests that create a DrawerHeader with only a child, or with no parameters at all.